src/controllers/product.controller.js

```diff
@@ -0,0 +1,29 @@
+const { productService } = require("../services");
+
+const getProduct = async (req, res) => {
+  try {
+    // const { page, sort, category } = req.query;
+    // console.log(queryParams);
+    // console.log(Object.entries(queryParams));
+    const userId = req.user_id;
+    const productList = await productService.getProductList(
+      userId
+      // page,
+      // sort,
+      // category
+    );
+    const totalProduct = await productService.getTotalProduct();
+    res.status(200).json({
+      message: "querySuccess",
+      data: productList,
+      productCount: totalProduct[0].product_count,
+    });
+  } catch (error) {
+    console.log("error", error);
+    res.status(error.status).json({ message: error.message });
+  }
+};
```

Comment:
```suggestion
  } catch (error) {
    console.log("error", error);
    res.status(error.status || 500).json({ message: error.message });
  }
};
```
예기치 못한 에러가 발생해서 `error.status`가 존재하지 않을 경우, error.status가 `undefined`가 되게 됩니다. 예기치 못한 에러라는 것은 서버 자체의 에러 발생이므로, 프론트엔드에게는 500으로라도 코드를 보내주시는 것이 좋습니다.